### DISCUSSION TITLE
通过矿神套件更新完之后，就启动不了了，不知道什么原因

### DISCUSSION THREAD

**Discussion by SYLRomance** [hslr-s/sun-panel#212] [Q&A]
RT，昨晚更新之前是正常使用的，但是通过矿神套件更新完之后，就启动不了了，卸载重装依旧无果，不知道什么原因

点“启动”，不超过10秒钟，就会自动关闭，且提示是“已手动停止”，查看日志也只有启动和停止信息，没有其他更多的信息了。。

求助！！！

**Comment by Tan-xiaozi** [comment]
这个不应该反馈到矿神那边么？矿神套件的适配好像作者并没有参与，假如无法使用，就是矿神那边没做适配，直接在矿神那边反馈就行了

**Comment by linllyw** [comment]
我也是这个问题，你解决了吗？如何处理呀？
